test/Engine/ProtoTest/Associative/BuiltinMethodsTest.cs

```diff
@@ -204,8 +204,8 @@ public void BIM14_Sort()
         //Test "SortIndexByValue()"
         public void BIM15_SortIndexByValue()
         {
-            String code =
-@"a = {1,3,5,7,9,8,6,4,2,0};
b = {1.3,2,0.8,2,null,2,2.0,2,null};
x = SortIndexByValue(a);
x1 = SortIndexByValue(a,true);
x2 = SortIndexByValue(a,false);
y = SortIndexByValue(b);
p = x[0];
p1 = x1[0];
p2 = x2[0];
q = x[9];
s = y[0];
t = y[7];
";
```

Comment:
@lukechurch yea ddnt realize it was this bad. Ive tried redoing the same file and spits out the same diffs. This is mainly die to the number of changes in a file.
What Ill do is Ill put in the main section above a few ported tests that highlight the changes of the entire PR. I dont think I can work around and do somthing about these diffs